test/e2e/bootstrap_resources.py

```diff
@@ -18,7 +18,7 @@
 from acktest.resources import read_bootstrap_config
 from e2e import bootstrap_directory
 
-SAGEMAKER_SOURCE_DATA_BUCKET = "source-data-bucket-718865417152-us-west-2"
+SAGEMAKER_SOURCE_DATA_BUCKET = "source-data-bucket-592697580195-us-west-2"
```

Comment:
Still getting the same error when using this bucket. `botocore.exceptions.ClientError: An error occurred (403) when calling the HeadObject operation: Forbidden
`

Reply:
try now

Reply:
Successfully created and copied data from that bucket on my end but looks like some tests are erroring out.

Reply:
Looked into it and looks like the s3 bucket you are providing doesn't have validation data for training jobs under `s3://$SAGEMAKER_DATA_BUCKET/sagemaker/training/validation`

Reply:
checking ... maybe the bucket I am copying from doesn't have the latest copy then